optimum/intel/neural_compressor/quantization.py

```diff
@@ -35,101 +40,292 @@
     AutoModelForSequenceClassification,
     AutoModelForTokenClassification,
     AutoModelForVision2Seq,
+    DataCollator,
+    PretrainedConfig,
+    PreTrainedModel,
     XLNetLMHeadModel,
+    default_data_collator,
 )
 from transformers.models.auto.auto_factory import _get_model_class
 from transformers.utils import TRANSFORMERS_CACHE, is_offline_mode
-from transformers.utils.versions import require_version
 
 import neural_compressor
-from huggingface_hub import hf_hub_download
+from huggingface_hub import HfApi, hf_hub_download
 from neural_compressor.adaptor.pytorch import PyTorch_FXAdaptor, _cfg_to_qconfig, _propagate_qconfig, get_torch_version
 from neural_compressor.adaptor.torch_utils.util import get_embedding_contiguous
-from neural_compressor.conf.config import Quantization_Conf
-from neural_compressor.experimental import Quantization
-from neural_compressor.utils.pytorch import _load_int8_orchestration
+from neural_compressor.model.torch_model import PyTorchModel
+from neural_compressor.quantization import fit
+from neural_compressor.utils.pytorch import _load_int8_orchestration, load
```

Comment:
shall we remove _load_int8_orchestration? it's an internal func and shouldn't use because it may be changed in the future in INC.

Reply:
Good catch ! it's not used, I'll remove this import